src/components/results.js

```diff
@@ -0,0 +1,54 @@
+import React, {Component} from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+
+class ResultSuccess extends Component {
+    render() {
+        return (
+        <View style={styles.container}>
+            <Text style={styles.text}>Hello Results Screen!</Text>
+        </View>
+    )}
+}
+
+class WikiDidMiss extends Component {
+    render() {
+        return (
+        <View style={styles.container}>
+            <Text style={styles.text}>Hello wikiMiss Screen!</Text>
+        </View>
+    )}
+}
+
+
+class BarcodeDidMiss extends Component {
+    render() {
+        return (
+        <View style={styles.container}>
+            <Text style={styles.text}>Hello barcodeMiss Screen!</Text>

```

Comment:
Functionally, this will be the same as the database miss screen. This is because we will be adding missed items to our database either way, and will not be adding to barcode databases either way.

Reply:
It is okay to have it built so that two separate screens could be shown, but for now we should only design the one and point both to that.